technical-reports/resolver/terminology.md

```diff
@@ -1,103 +1,37 @@
-# Resolver Terminology
+# Terminology
 
-<dl>
-<dt>Resolver</dt>
-<dd>
+## <dfn>Resolver</dfn>
 
-A mechanism that combines token sets and applies modifiers to produce a final, resolved set of design tokens.
+The mechanism by which multiple possible values of design tokens are reduced to a single value, i.e. this module.
 
-</dd>
-<dt>Token Set</dt>
-<dd>
+## <dfn>Set</dfn>
 
-A collection of design tokens grouped together, often representing foundational tokens, component-specific tokens, or theme-specific tokens.
+Alternate design token values that are defined in [&lt;token-defs&gt;](#token-defs), intended to be merged in a specific order.
 
-</dd>
-<dt>Dimension</dt>
-<dd>
+## <dfn>Modifier</dfn>
 
-Dimensions are categories used to organize your design tokens. See them as contexts in which token values might change:
+Alternate design token value accessed with an [=input=], that takes priority over a [=set=]. Modifiers MAY be [=orthogonal=].
 
-- Brands
-- Surface
-- Language direction
-- Themes
-- Platform
-- Screen size
-- Density
-- Component
-- State
-- Variant
-- Contrast
+## <dfn>Input</dfn>
 
-<aside class="issue">
-
-The term "Dimension" is potentially confusing since "Dimension" is also a type of token and a general concept. Consider using "Context" instead, which aligns with the existing definition as "contexts in which token values might change." Additionally, explicit mention of color schemes may be problematic - developers often distinguish between "themes" and "color modes" (like Windows High Contrast mode for accessibility), and it's unclear whether everyone would understand themes to encompass dark/light/high-contrast modes.
-
-</aside>
-
-</dd>
-
-<dt>Modifier</dt>
-
-<dd>
-
-An entity that modifies or overrides tokens in the base sets. Modifiers can represent dimensions like themes, modes, brands, or any other contextual variations.
-
-<dl>
-
-<dt>Enumerated Modifier</dt>
```

Comment:
All the _types_ of modifiers here aren’t “terminology” per se, I think they should either be expressed as **Syntax** or outright removed.